src/main/scala/cromwell/binding/WdlExpression.scala

```diff
@@ -44,7 +45,7 @@ object WdlExpression {
           throw new WdlExpressionException("RHS must be an identifier")
         }
         val rhs = rhsAst.asInstanceOf[Terminal].getSourceString
-        lhs.value.asInstanceOf[WdlObject].getOrElse(rhs, {
+        lhs.asInstanceOf[WdlObject].getOrElse(rhs, {
```

Comment:
I'm also wondering if there's some way around all of these asInstanceOfs to make it less runtime explosive, hmmmmmm